.circleci/config.yml

```diff
@@ -5,47 +5,32 @@ orbs:
 
 jobs:
 
-  build-11:
+  build:
+    description: Build the citus extension
+    parameters:
+      postgres:
```

Comment:
Minor: to remove the mental mapping, we can call this parameter `postgres_major` or `pg_major`